core/crop/structure.py

```diff
@@ -27,7 +27,7 @@
 
 from bcrypt import gensalt, hashpw
 
-from crop.constants import (
+from core.crop.constants import (
```

Comment:
Instead of adding **core.**, could you please adjust your python path since we have restructured the repository.